documentation/code-reviews.md

```diff
@@ -0,0 +1,67 @@
+# How we review code
+
+We consider code reviews an essential practice of software engineering. Following the collective refinement of the task and one of the team taking up its implementation, the code review is the moment the work becomes collaborative again.
+
+You open a pull request, explaining your approach and choices. Peers invest time in understanding the attempt and providing feedback. We document any emerging guidelines or decisions, do the work to resolve all comments, and integrate the increment into the product.
+
+## Write a pull request
+
+If you believe that the code in your branch meets the applicable acceptance criteria and the definition of done, you create a pull request (PR). If you have just pushed, a handy link will appear on Azure’s ‘Pull requests’ page to create one immediately. Otherwise, you configure the new PR yourself.
```

Comment:
```suggestion
If you believe that the code in your branch meets the applicable acceptance criteria and the definition of done, you create a pull request (PR). If you have just pushed, a handy link will appear in the repo in github.com to create one immediately. Otherwise, you configure the new PR yourself.
```